generator/lib/jbuild

```diff
@@ -0,0 +1,32 @@
+(* -*- tuareg -*- *)
+#require "unix"
+
+let flags = function
+  | [] -> ""
+  | pkgs ->
+    let cmd = "ocamlfind ocamlc -verbose" ^ (
+        List.fold_left (fun acc pkg -> acc ^ " -package " ^ pkg) "" pkgs
+      ) in
+    let ic = Unix.open_process_in
+        (cmd ^ " | grep -oEe '-ppx (\"([^\"\\]|\\.)+\"|\\w+)'")
+    in
+    let rec go ic acc =
+      try go ic (acc ^ " " ^ input_line ic) with End_of_file -> close_in ic; acc
+    in
+    go ic ""
+
+let flags = flags ["ppx_deriving_rpc"]
```

Comment:
This is no longer needed, you can remove all the autogeneration and add `(preprocess (pps (ppx_deriving_rpc)))` if `ppx_type_conv` or `ppx_sexp_conv` is not used. After `opam-next` is merged it will be no longer needed at all

Reply:
`opam-next` has been merged, we could have a test withouth the `flags` if you want

Reply:
Ok, I'm testing it.